src/main/resources/templates/header.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Header</title>
+    <meta charset="UTF-8">
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css'>
+    <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css'>
```

Comment:
Maybe you should create a fragment for this in your `common.html` as well.

Reply:
Fragment created for all the CSS and JS files